Book - Handling Input Chapter & Initial Page (#2417)
bevyengine/bevy-website: content/learn/book/handling-input/_index.md

Reviewer: Hm. Not specific to this chapter: For the multi-page chapters, the introduction page doesn't contain much useful information. I see two things it's attempting to do:
- Summarize what the chapter is about
- Summarize what the problem is/how Bevy is approaching it

The first is already mostly served by the chapter & subchapter names. The second one is on a very superficial level where one has to read the chapter to learn anything useful. The chapter introductions in the [Rust book](https://doc.rust-lang.org/book/) seem to have the same goals but work better as they tend to go into more detail.

In this case the content of the last sentence of this page is just a duplication of the first sentence of the next page. So this page says very little. I would prefer we move the content of the "Using Input" page directly here.

Author: I was thinking about this as well, but I would prefer to make each `_index` page a chapter summary (like a "go to _x page_ for _x information_ and _y page_ for _y information_") rather than making it an introduction page. 

When I was first going through the book, I didn't even realize that the `_index` pages existed and had writing on them since they have the option to drop down a list of the individual pages that have the actual information I want. Maybe it's just me being a bit dense, but I can't imagine I'm the only one who would encounter this. I think avoiding placing information in areas that people might not be able to intuitively find should be a higher consideration than potentially having duplicate information.

For this chapter's `_index` page specifically, I am not opposed to changing what's on there (especially if we do want to make the `_index` a summary that links out to each subpage), ~~but any changes I think should be done as part of a separate PR (or multiple if we're changing them to summaries)~~.

Author: Of course I say this before I start making changes to `_index`, ignore that last section

+++
title = "Handling Input"
template = "docs.html"
insert_anchor_links = "right"
[extra]
weight = 14
status = 'hidden'
+++

By definition, games are interactive.
Players will perform some action or reaction, and the game will react and change in return.
When it comes to video games specifically, we need to receive input from the player through some kind of device.

> Whether it's controllers, keyboards, mice, or even touch screens, Bevy allows us to interact with input from all of these devices.

Reviewer: ```suggestion
When it comes to video games specifically, we need to receive input from the player through some kind of device:
Controllers, keyboards, mice, or even touch screens.
```
It seems to me like the last sentence part was superfluous: If Bevy didn't allow us to interact with some of these devices, we would either say that or not discuss them in the first place.

+++
title = "Handling Input"
template = "docs.html"
insert_anchor_links = "right"
[extra]
weight = 14
status = 'hidden'
+++

> By definition, games are interactive.

Reviewer 2: ```suggestion
If a player is to *play* your game, it must be interactive.
```

Players will perform some action or reaction, and the game will react and change in return.

> When it comes to video games specifically, we need to receive input from the player through some kind of device.

Reviewer 2: ```suggestion
To make this happen, we need to receive input from the player through some kind of device.
```

> In Bevy, we can use input from gamepads, keyboards, mice, or even touch screens in our games and applications.

Reviewer 2: ```suggestion
In Bevy, we can collection input from gamepads, keyboards, mice, or even touch screens.
```

The process of accessing and using this data is detailed on the [Using Input page](/learn/book/handling-input/using-input), which is where we'd recommend you start.

> While input data from every device will follow the same process, you should also read the page of each device relevant to your project to see how their data is uniquely handled:

Reviewer 2: Cut from this PR: add later.

- [Keyboard Input](/learn/book/handling-input/keyboard-input)
- [Mouse Input](/learn/book/handling-input/mouse-input)
- [Gamepad Input](/learn/book/handling-input/gamepad-input)
- [Touch Input](/learn/book/handling-input/touch-input)

> When you use larger groupings of features (like profiles and collections), all of these devices are enabled in your game by default.

Reviewer 2: Cut from this PR. I don't think this is worth discussing.
